mlflow/langchain/__init__.py

```diff
@@ -882,6 +883,33 @@ def load_model(model_uri, dst_path=None):
     return _load_model_from_local_fs(local_model_path)
 
 
+def _patch_runnable_cls(cls, patched_class):
+    if cls.__name__ not in patched_class:
```

Comment:
can we do this check in `if cls.__name__ not in patched_class:`?

Reply:
Moved inside _inspect_module_and_patch_cls